tests/integration_test.rs

```diff
@@ -4,77 +4,175 @@ use tokio::net::TcpStream;
 use tokio::stream::StreamExt;
 use tokio_util::codec::{Framed, LengthDelimitedCodec};
 
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 
 use bytes::Bytes;
 
 static LOCAL_REMITS: &str = "localhost:4242";
+static OK_RESP: &[u8] = b"ok";
 
-macro_rules! should_respond_with {
-    ($framer:expr, $bytes:expr, $resp:expr) => {
-        $framer
-            .send(Bytes::from($bytes))
-            .await
-            .expect("could not send command");
+#[tokio::test]
+async fn integration_tests() {
+    let ref mut framer = connect_to_remits().await;
+
+    println!("test: should be able to add a log");
```

Comment:
Since to see these printlns you would have to do `cargo test -- --nocapture` maybe just make them comments or make a makefile so you can just run `make test` or w/e

Reply:
They show if there's a panic/one of the conditions fails.  So they show when we need them.  I still believe that we should move the whole thing out of cargo test, but I didn't wanna rabbit hole too much while trying to get the other stuff out.